presubmit.yaml

```
@@ -4,3 +4,6 @@ steps:
     script: |
       go build -C custom-targets/util/applysetters
       go build -C custom-targets/util/clouddeploy
+      go build -C custom-targets/git-ops/git-deployer
```

Can you add the docker build as well?

```suggestion
  - name: docker
    script: |
        docker build custom-targest/git-ops/git-deployer
```

---

Done